language/move-lang/src/parser/syntax.rs

```diff
@@ -417,7 +457,7 @@ fn parse_attribute_value(tokens: &mut Lexer) -> Result<AttributeValue, Error> {
         return Ok(sp(v.loc, AttributeValue_::Value(v)));
     }
 
-    let ma = parse_module_access(tokens, || "attribute name value".to_string())?;
+    let ma = parse_name_access_chain(tokens, || "attribute name value")?;
```

Comment:
The grammar comment for `AttributeValue` should refer to `NameAccessChain` instead of `ModuleAccess`